numpy/ma/tests/test_deprecations.py

```diff
@@ -0,0 +1,41 @@
+"""Test deprecation and future warnings.
+
+"""
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+from numpy.testing import TestCase, run_module_suite, assert_warns
+from numpy.ma.testutils import assert_equal
+
+
+class TestArgsort(TestCase):
+	""" gh-8701 """
```

Comment:
Should include date and numpy version, that helps keep track of expiration dates, etc.